compile_matlab_bindings_for_precice.m

```diff
@@ -5,7 +5,7 @@
 % from a different folder
 path = string(fileparts(mfilename('fullpath')));
 path_Interface = strjoin([path,"+precice","@SolverInterface","private","preciceGateway"],filesep);
-path_Constants = strjoin([path,"+precice","@Constants","private","preciceGateway"],filesep);
+path_Constants = strjoin([path,"+precice","+constants","private","preciceGateway"],filesep);
```

Comment:
```suggestion
path_namespace_constants = strjoin([path,"+precice","+constants","private","preciceGateway"],filesep);
```

Reply:
This will require some more renames downstream. Alternatively, one can also change `path_namespace_precice`. In the end, the most important this is consistency.